frontend/public/components/utils/useEventListener.jsx

```diff
@@ -0,0 +1,26 @@
+import { useRef, useEffect } from 'react';
+
+const useEventListener = (eventName, handler, element = window) => {
+  const updatedHandler = useRef();
+
+  useEffect(() => {
+    updatedHandler.current = handler;
+  }, [handler]);
+
+  useEffect(() => {
+    const isSupported = element && element.addEventListener;
```

Comment:
The `addEventListener` DOM API is supported by all modern browsers ([link](https://caniuse.com/#search=addeventlistener)).

As written in the main `README` file, Console supports the latest version of
- Edge
- Chrome
- Safari
- Firefox

so I'd suggest not checking for `addEventListener` support and just assume that it is supported.

Reply:
fixed